bobtimus/src/lib.rs

```diff
@@ -154,40 +148,52 @@ impl<R, RS> Bobtimus<R, RS> {
                             })?
                             .clone();
 
-                        Result::<_, anyhow::Error>::Ok((txin, txin_as_txout, blinding_key))
+                        Result::<_, anyhow::Error>::Ok(swap::Input {
+                            tx_in,
+                            tx_out,
```

Comment:
:upside_down_face: Super nitpicky, but we almost exclusively use `txin` and `txout` in this workspace.